src/shchgrp.c

```diff
@@ -0,0 +1,91 @@
+/* shchgrp
+ *
+ * SHitty
+ * CHained
+ * GReP
+ * 
+ * Filter text with multiple regexes. It's an over-engineered way to chain a few
+ * regexes together. 
+ * 
+ * Written by Seungin Lyu, Adam Novotny, Matthew Beaudouin-Lafon
+ */
+
+#include "shvec.h"
+#include "regexlib.h"
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <argp.h>
+
+int arg_parse(int argc, char* argv[]){
+    // parse arguments
+    char ch;
+    while((ch = getopt(argc, argv, "")) != EOF){
+        switch (ch){
+            default:
+                fprintf(stderr, "Error: Unknow Flag");
+                return 1;
+        }
+    }
+    return 0;
+}
+int main(int argc, char *argv[]){
+    int num_regexes = 0;
+
+    Regex ** regexes;
+    int * matches;
```

Comment:
We should do a pass for unifying pointer declaration syntax. At a glance we have all combinations of space before and after the *.